standard/element_parameter.md

```diff
@@ -0,0 +1,10 @@
+# Element Parameter Naming
+
+Lattice elements parameters are organized into **parameter groups**. All groups are organized as abstract syntax trees.
+At the top level, there are the groups with names like `MagMultipoleGroup`, `ElecMultipoleGroup`, `RFGroup, `BendGroup`, `AlignmentGroup`, etc. 
+Group names use Upper camel case and have `Group` at the end of the name.
+
+A dot `.` is used as a separator between levels in a group. 
+For example, `BendGroup.angle`, `BendGroup.e1`, `RFGroup.frequency`, etc. Using lower case for second and lower level names is encouraged but not mandated.
+
+There are two element parameters that are so common that they are not grouped. These element parameters are `L` (element length) and `Name` (name of element).
```

Comment:
Apertures are also common to every element in the lattice.
At the top, there should be a list of constants, such as the length of the lattice (ring =circumference, radius). For the group elements, as well as the length and aperture, maybe the strengts.

Reply:
Apertures are common to most elements but there can be elements that don't have apertures. Fiducial markers for example. 

The PR here is just a start on the section of the standard that describes element attributes. Stuff like specifying the lattice length would go elsewhere. So I do not see anything to change here. what are your thoughts?